### PR TITLE
Added image link to projects

### PR DESCRIPTION
**Pull request by alexander-astrand**

### CODE REVIEW

File: cosmwasm/contracts/wattpeak-minter/src/query.rs [hunk 1]
```diff
@@ -179,18 +200,26 @@ mod tests {
...
             }).unwrap();
+            
             let project: crate::state::Project = from_json(&res).unwrap();
             assert_eq!(project.name, "test name");
             assert_eq!(project.description, "test description");
             assert_eq!(project.document_deal_link, "ipfs://test-link");
             assert_eq!(project.max_wattpeak, 1000);
             assert_eq!(project.minted_wattpeak_count, 0);
+            assert_eq!(project.image_link, "ipfs://test-image");
+            assert_eq!(project.location.latitude, Decimal::from_ratio(1001u64, 10u64));
```
Comment (gjermundgaraba) [inline cosmwasm/contracts/wattpeak-minter/src/query.rs:222]: Check the longitude as well.

Could you also do a verification of what the string representation would look like for both latitude and longitude, so we are sure it comes out as expected to the client?

File: cosmwasm/contracts/wattpeak-minter/src/state.rs [hunk 2]
```diff
@@ -2,6 +2,8 @@ use cosmwasm_schema::cw_serde;
 use cosmwasm_std::{Addr, Coin, Decimal, Deps, StdError, StdResult};
 use cw_storage_plus::{Item, Map};
 
+use crate::msg::Location;
```
Comment (gjermundgaraba) [inline cosmwasm/contracts/wattpeak-minter/src/state.rs:5]: The state should not depend on the msg. If anything it should be the other way around. Put the location struct here and depend on it in the msg

**Review by gjermundgaraba — CHANGES_REQUESTED** [review]

File: cosmwasm/contracts/wattpeak-minter/src/query.rs (hunk 1 above)
Comment (alexander-astrand) [reply cosmwasm/contracts/wattpeak-minter/src/query.rs:224]: Sorted!

File: cosmwasm/contracts/wattpeak-minter/src/state.rs (hunk 2 above)
Comment (alexander-astrand) [reply cosmwasm/contracts/wattpeak-minter/src/state.rs]: Done!

**Review by gjermundgaraba — APPROVED** [review]

**Pull request merged**